Backend/.gitignore

```diff
@@ -609,5 +609,3 @@ msbuild.wrn
 **/net7.0/
 **/bin/
 **/.vs/
-
-**/Migrations/
```

Comment:
This shouldn't have been changed. You can manualy add the removed lines at the end of the file and do one more commit